src/NServiceBus.Core/Routing/DistributionContext.cs

```diff
@@ -0,0 +1,49 @@
+namespace NServiceBus.Routing
+{
+    using System.Collections.Generic;
+    using Extensibility;
+    using Pipeline;
+
+    /// <summary>
+    /// The distribution context for the DistributionStrategy implementers.
```

Comment:
maybe `The context for custom <see cref="DistributionStrategy"/> implementations` ?